llmfoundry/data/packing.py

```diff
@@ -106,7 +106,7 @@ def pack(self, batch: dict[str, torch.Tensor]) -> dict[str, torch.Tensor]:
                 'sequence_id',
             ]
         # Cut everything down to size
-        sizes, trimmed_examples = _trim_batch(batch)
+        sizes, trimmed_examples = BinPackCollator._trim_batch(batch)
```

Comment:
Can you do `self._trim_batch`?

Reply:
Maybe we need classmethod for this to work properly with inheritance?

I should be able to create a class that only replaces the trim batch function:
```
class MyBinPackCollator(BinPackCollator):
     def _trim_batch(...):
           # my own logic

```